performance.rst

```diff
@@ -81,7 +81,7 @@ container into a single file, which could improve performance when using
         // config/services.php
 
         // ...
-        $container->parameters()->set('.container.dumper.inline_factories', true);
+        $configurator->parameters()->set('.container.dumper.inline_factories', true);
```

Comment:
I guess the whole example needs to look like this:

```suggestion
        namespace Symfony\Component\DependencyInjection\Loader\Configurator;

        return function(ContainerConfigurator $container): void {
            $container->parameters()->set('.container.dumper.inline_factories', true);
        };
```

Reply:
I've done this change while merging. Thanks!